src/components/users/SignUpForm.js

```diff
@@ -89,6 +91,7 @@ const SignUpForm = () => {
         inputName="password1"
         inputValue={password1}
         inputOnChange={handleChange}
+        inputPlaceHolder="MIN. 8 CHARACTERS LONG"
```

Comment:
get this from `en.js`

Reply:
If I put `<FormattedMessage id="userform.password.placeholder.label.text" />` then I don't see the message. I see something like `object.Object`. I need to put a raw string there. Do you know why?

Reply:
I think it should be something like: `placeholder={intl.formatMessage({ id: 'generic.to' })}`